app/models/journal.rb

```diff
@@ -0,0 +1,3 @@
+class Journal < ApplicationRecord
+  belongs_to :user
```

Comment:
Is this correct? You have defined `has_many :journals` in `client`, not in `user`.